### ISSUE TITLE
Auto-sync included ignore file

### ISSUE THREAD

**Issue by xeruf** [syncthing/syncthing#9352]
Currently, when setting up a new device, it goes something like this:
1. add the folder
2. manually setup an ignore rule that ignores all files except the ignore-file I need
3. wait for the ignore-file to be synced
4. include the ignore-file in .stignore and let the sync proceed

Step 2+3 are quite annoying and could be eliminated if syncthing simply tried to fetch an included ignore file from the remotes before throwing an error. I don't see how this would be problematic in any way, and would greatly ease my setup :)

**Comment by imsodin** [comment]
Yep, I want that too :)  
I thought we have a ticket for it already, but I couldn't find it.

I once had a look at it. As far as I remember it requires splitting up the pulling code into two stages: One to discover what there is to pull, and one to act on it. And then expose the second part and use it to pull included ignore files when they are missing. Probably all but trivial, but if it can be done without creating a huge mess I think it would be very worth it.

**Comment by Jakobohrnell** [comment]
Any effort being put in to this? Am having to get up to a bunch of shenanigans with other tools just to circumvent this issue.

**Comment by DigitalCyberSoft** [comment]
I don't mind that #10668 was closed as a duplicate, but are there any thoughts on the expressed implementation?

**Comment by calmh** [comment]
Yeah, I have no desire to try to interpret that AI fever dream, but it poked me into a simple, alternative solution in #10669.
